templates/compose/evolution-api.yaml

```
@@ -137,13 +139,27 @@ services:
+      interval: 10s
+      timeout: 5s
+      retries: 30
+      start_period: 60s
+
   redis:
     image: 'redis:latest'
     command: "redis-server --port 6379 --appendonly yes\n"
-    restart: always
     volumes:
       - 'evolution_redis:/data'
+    image: 'redis:7-alpine'
```

YAMLlint (1.37.1)

[error] duplication of key "image" in mapping

(key-duplicates)